.git-blame-ignore-revs

```diff
@@ -0,0 +1 @@
+dc9c80d4d1142d5de37c133c15261fd1d006dcba
```

Comment:
if you push to fix the comment below, you will have to change this commit id.
When merging, the maintainer/merger should make sure that the commit id is not changed.